src/providers/zig.rs

```diff
@@ -0,0 +1,80 @@
+use crate::nixpacks::{
+    nix::pkg::Pkg,
+    phase::{BuildPhase, InstallPhase, SetupPhase, StartPhase},
+};
+
+use super::Provider;
+
+pub struct ZigProvider;
+
+impl Provider for ZigProvider {
+    fn setup(
+        &self,
+        _app: &crate::nixpacks::app::App,
+        _env: &crate::nixpacks::environment::Environment,
+    ) -> anyhow::Result<Option<crate::nixpacks::phase::SetupPhase>> {
```

Comment:
Replace `anyhow::Result` with `Result` to be consistent with rest of the providers.
```suggestion
    ) -> Result<Option<crate::nixpacks::phase::SetupPhase>> {
```